### PR TITLE
weather app rainbow style

### PR DESCRIPTION
**Pull request by nathalienordstrom**
Link to my deployed project:
https://weatherrainbow.netlify.app
My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: Designs/Design-1/style.css [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+  margin:0;
+  font-size: 30px;
+}
+
+.forecast-minmax {
+  margin:0;
+  font-size: 35px;
+}
+}
+
```
Comment (Snabbina) [inline Designs/Design-1/style.css:292]: some inconsistency in the spacing between your blocks of code and some curly-brackets.

**Review by Snabbina — COMMENTED** [review]
YOU DID IT NATHALIE! Its a real weather-rainbow app🌦 . Your code was easy to follow and has the necessary components to create and transform the data into a visual weather app. You have worked with responsiveness and started from mobile first! Super good! For the future watch out for to many empty spaces between code blocks and maybe having more of a habit of commenting more in your code (even in html or css if needed) , for even more readability, as the code will become more and more complex. Well done, keep hanging in there!🧘🏽‍♀️🏋🏽‍♀️

File: Designs/Design-1/.vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
[the remark this answers is not in the document]
Comment (EmOlsz) [reply Designs/Design-1/.vscode/settings.json:3]: Nope, whole hidden folder `.vscode` should be included to `.gitignore` file. Needs to be fixed.
At this point you need to do it like this :

1. On github (inside browser) delete `.vscode` from your repo.
2. On local version of project (on your hard drive) add file `.gitignore`
3. Inside that file, add separate lines `.DS_Store` and `.vscode`
4. Commit and push your changes

File: Designs/Design-1/index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        <section class="today">
+            <h1>Today</h1>
+
+            <div id="degree"></div>
+            <div id="city"></div>
+            <div id="weather"></div>
+            <div id="feels"></div>
+            <div class="sunny">
+                <div id="sunrise"></div>
+                <div id="sunset"></div>
```
Comment (EmOlsz) [inline Designs/Design-1/index.html:27]: You missed one closing tag somewhere here - needs to be fixed

File: Designs/Design-1/index.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+            </div>
+    </div>
+    </section>
+
+    </div>
+
+    <script src="./index.js"></script>
+</body>
+
+</html>
```
[the remark this answers is not in the document]
Comment (EmOlsz) [reply Designs/Design-1/index.html:53]: Even should

File: Designs/Design-1/index.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+const weather = document.getElementById("weather");
+const feels = document.getElementById("feels");
+
+const sunrise = document.getElementById("sunrise");
+const sunset = document.getElementById("sunset");
+
+
+
+
+
```
Comment (EmOlsz) [inline Designs/Design-1/index.js:15]: A lot of empty lines, let's delete them as well ;)

File: Designs/Design-1/index.js [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+        hour12: false
+    });
+    return readableTime;
+}
+
+fetch(currentUrl)
+    .then((response) => {
+        return response.json()
+    })
+
```
Comment (EmOlsz) [inline Designs/Design-1/index.js:31]: Unnecessary blank line

File: Designs/Design-1/index.js [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+    .then((json) => {
+        console.log(json)
```
Comment (EmOlsz) [inline Designs/Design-1/index.js:33]: Before pushing, we should delete all `console.log()` occurrences

File: Designs/Design-1/style.css [hunk 8]
```diff
@@ -0,0 +1,292 @@
+body {
+  font-family: 'Lato', sans-serif;
+  margin: 0;
+  text-shadow: 1px 1px 1px black;
+  color:white;
+  background-image: linear-gradient(to top, rgb(247, 151, 151),rgb(247, 213, 150),rgb(247, 247, 172),rgb(201, 245, 201),rgb(172, 172, 244),rgb(210, 152, 251),rgb(251, 220, 251)); 
+}
+
+
```
Comment (EmOlsz) [inline Designs/Design-1/style.css:9]: too many blank lines

File: Designs/Design-1/style.css [hunk 9]
```diff
@@ -0,0 +1,292 @@
...
+  align-items: center;
+  color:white;
+}
+h1 {
+  font-size:40px;
+  margin-top:20px;
+  margin-bottom:0;
+}
+
+#degree{
```
Comment (EmOlsz) [inline Designs/Design-1/style.css:22]: We should not style by id, classes are far better

File: Designs/Design-1/style.css [hunk 10]
```diff
@@ -0,0 +1,292 @@
...
+  font-size: 60px;
+  font
```
Comment (EmOlsz) [inline Designs/Design-1/style.css:24]: ? ;)

File: Designs/Design-1/style.css [hunk 11]
```diff
@@ -0,0 +1,292 @@
...
+  position: relative;
+  height: 30px;
+  width: 100%;
+  background: linear-gradient(white, pink, lightblue);
+  animation: rotate 1.5s linear infinite;
+  }
+  
+.outer-line span {
+  position: absolute;
+  left: 0px;
```
Comment (EmOlsz) [inline Designs/Design-1/style.css:77]: If value is `0px`, we can omit `px` and just write `0`

File: Designs/Design-1/style.css [hunk 12]
```diff
@@ -0,0 +1,292 @@
...
+    padding-top:100px;
+  }
+  .sunny {
+    display: flex;
+    flex-direction: column;
+    align-items: center;  
+    font-size: 40px;
+  }
+  
+img {
```
Comment (EmOlsz) [inline Designs/Design-1/style.css:201]: Wrong indentation

**Review by EmOlsz — CHANGES_REQUESTED** [review]
Congratulations on your working app Nathalie! I'm very happy to see it, truly :) I hope it will get better and better with each week from now on, fingers crossed for that :)
Nonetheless, there is a lot of minor issues in your code (mostly semantical). Please, take a look on my comments and provide changes. Note that I haven't pointed out every place were you made mistake, I chose only one example of each issue.
After you will fix your code, I will approve project :)

File: Designs/Design-1/.vscode/settings.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (nathalienordstrom) [reply Designs/Design-1/.vscode/settings.json]: I don´t know the commands for that?

**Review by EmOlsz — APPROVED** [review]

**Review by EmOlsz — APPROVED** [review]

**Pull request closed without merging**